etc/kayobe/ansible/fix-hostname.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- name: Fix hostname on storage nodes for cephadm
-  hosts: storage
+- name: Fix hostname
+  hosts: reboot_hosts
```

Comment:
This group doesn't really match the name or content of the playbook

Reply:
Somehow managed to write that three times without realising it was wrong. Fixed now